cmd/authgear/importer/importer.go

```diff
@@ -0,0 +1,189 @@
+package importer
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/authgear/authgear-server/pkg/lib/authn/identity/loginid"
+	"github.com/authgear/authgear-server/pkg/lib/config"
+	"github.com/authgear/authgear-server/pkg/lib/infra/db"
+	"github.com/authgear/authgear-server/pkg/lib/infra/db/appdb"
+	"github.com/authgear/authgear-server/pkg/util/validation"
+)
+
+type Importer struct {
+	AppID         config.AppID
+	Handle        *appdb.Handle
+	SQLBuilderApp *appdb.SQLBuilderApp
+	SQLExecutor   *appdb.SQLExecutor
+	EmailConfig   *config.LoginIDEmailConfig
+}
+
+func (i *Importer) ImportFromCSV(csvPath string) error {
+	now := time.Now().UTC()
+
+	f, err := os.Open(csvPath)
+	if err != nil {
+		return err
+	}
+
+	headerSkipped := false
+	r := csv.NewReader(f)
+	for {
+		record, err := r.Read()
+		if errors.Is(err, io.EOF) {
+			break
```

Comment:
If we are going to have report summary, we need to add this row to the problematic rows.